src/eventHandlers/invocationRequest.ts

```diff
@@ -15,8 +16,8 @@ import LogLevel = rpc.RpcLog.Level;
  * @param requestId gRPC message request id
  * @param msg gRPC message content
  */
-export function invocationRequest(channel: WorkerChannel, requestId: string, msg: rpc.InvocationRequest) {
-    const info = channel.functionLoader.getInfo(msg.functionId);
+export function invocationRequest(channel: WorkerChannel, requestId: string, msg: rpc.IInvocationRequest) {
+    const info = channel.functionLoader.getInfo(nonNullProp(msg, 'functionId'));

```

Comment:
I don't necessarily see the benefit of generalizing to the interface, especially when the class avoids the need for using `nonNullProp` here?

Reply:
If I leave it as `InvocationRequest`, I get this error in `setupEventStream.ts`:

> Argument of type 'IInvocationRequest' is not assignable to parameter of type 'InvocationRequest'.
  Property 'toJSON' is missing in type 'IInvocationRequest' but required in type 'InvocationRequest'.

So this was really just a byproduct of my code in `setupEventStream` being more type-safe than the old code in `WorkerChannel.ts`. Essentially, there's no guarantee we'll get an instance of the class here which is why TypeScript threw the error

Also, `nonNullProp` is a little util my old teams and I used pretty regularly so I expect it to be more and more prevalent in our codebase and not much of a "downside"